.changeset/weak-chairs-add.md

```diff
@@ -0,0 +1,5 @@
+---
+"@justeattakeaway/pie-modal": minor
+---
+
+[Added] prop to control modal open & close state
```

Comment:
Super small NIT for future PRs – should be a `-` between the `[Added]` and description 😄